packages/cloudcasa/generated-changes/patch/Chart.yaml.patch

```diff
@@ -1,21 +1,20 @@
 --- charts-original/Chart.yaml
 +++ charts/Chart.yaml
-@@ -5,7 +5,7 @@
- kubeVersion: ">=1.13.0-0"
+@@ -6,7 +6,8 @@
  description: CloudCasa backup service for Kubernetes and cloud native applications
  home: https://cloudcasa.io
--icon: file://../logo.png
+ #icon: file://../logo.png
+-icon: https://raw.githubusercontent.com/catalogicsoftware/cloudcasa-helmchart/gh-pages/logo.png
++#icon: https://raw.githubusercontent.com/catalogicsoftware/cloudcasa-helmchart/gh-pages/logo.png
```

Comment:
Can you remove those comments to avoid cluttering the patch file? It should be okay to just replace it with the new URL.